e2e_tests/tests/experiment/test_unmanaged.py

```diff
@@ -28,7 +28,7 @@ def test_unmanaged() -> None:
 @pytest.mark.timeout(10)
 def test_unmanaged_termination() -> None:
     # Ensure an erroring-out code does not hang due to a background thread.
-    exp_path = EXAMPLES_ROOT / "1_error.py"
+    exp_path = conf.fixtures_path("unmanaged/error_termination.py")
```

Comment:
I assume that'll require a mock master server with all APIs used by harness in detached mode. I think that's too much effort for little benefit.

Reply:
hm, ok.

So maybe outside the scope of this PR then, but can you set up tensorboard uploading to s3 with an otherwise-dummy core context?  It seems like the master shouldn't be required to exercise this codepath.